brave/browser-laptop
# Save the window size (width, height) and use it when restoring from s…

Save the window size (width, height) and use it when restoring from session

Fixes https://github.com/brave/browser-laptop/issues/4385

Auditors: @bridiver @brendaneich

Test Plan:
1) Open 2 windows
2) Resize windows to be different sizes than each other. Maybe window 1 is 300 wide and 800 tall, window 2 is 600 wide and 400 tall
3) Quit Brave
4) Re-launch Brave
5) windows should be restored to the same position and shuld be the same size
